pkg/controller/controller_clusterserviceplan.go

```diff
@@ -26,37 +26,37 @@ import (
 	"k8s.io/client-go/tools/cache"
 )
 
-// Service plan handlers and control-loop
+// Cluster service plan handlers and control-loop
 
-func (c *controller) servicePlanAdd(obj interface{}) {
+func (c *controller) clusterServicePlanAdd(obj interface{}) {
 	key, err := cache.DeletionHandlingMetaNamespaceKeyFunc(obj)
 	if err != nil {
 		glog.Errorf("ClusterServicePlan: Couldn't get key for object %+v: %v", obj, err)
 		return
 	}
-	c.servicePlanQueue.Add(key)
+	c.clusterServicePlanQueue.Add(key)
 }
 
-func (c *controller) servicePlanUpdate(oldObj, newObj interface{}) {
-	c.servicePlanAdd(newObj)
+func (c *controller) clusterServicePlanUpdate(oldObj, newObj interface{}) {
+	c.clusterServicePlanAdd(newObj)
 }
 
-func (c *controller) servicePlanDelete(obj interface{}) {
-	servicePlan, ok := obj.(*v1beta1.ClusterServicePlan)
-	if servicePlan == nil || !ok {
+func (c *controller) clusterServicePlanDelete(obj interface{}) {
+	clusterServicePlan, ok := obj.(*v1beta1.ClusterServicePlan)
+	if clusterServicePlan == nil || !ok {
```

Comment:
unrelated to this PR, but I'm not sure if ignoring `!ok` (i.e. we get the object of unexpected type) is a good idea.